biz.aQute.repository/src/aQute/bnd/repository/p2/provider/P2Config.java

```diff
@@ -32,10 +32,4 @@
 	 */
 	String location(String string);
 
-	/**
-	 * If not set or false, this assumes a P2 repository, i.e. the url points to
-	 * a P2 repository directory. If set to true, the url is assumed to point to
-	 * an Eclipse Target platform.
-	 */
-	boolean targetPlatform();
```

Comment:
Removing this hardly seems worth a major package bump. Perhaps just mark it deprecated instead. And you don't need to document it in the docs page.

Reply:
We could deprecate it, but for how long should it stay in there? I've needed to bump the Version this hard, because baselining complained about it.

Reply:
It is complaining because you deleted the method from an interface in an exported package :-). So don't delete it. Just mark is deprecated and comment that it is unused. There is little harm in leaving the method in place.